ecs-cli/modules/config/params_test.go

```diff
@@ -156,6 +162,50 @@ func TestNewCliParamsWhenPrefixKeysAreNotPresent(t *testing.T) {
 	assert.Equal(t, ecscli.CFNStackNamePrefixDefaultValue+clusterName, params.CFNStackName, "Expected CFNStackName to be default")
 }
 
+func TestNewCliParamsWithAWSProfile(t *testing.T) {
+	// Keys in env vars take highest precedence; ensure they are not set
+	os.Unsetenv("AWS_ACCESS_KEY")
+	os.Unsetenv("AWS_SECRET_KEY")
+
+	configContents := `[awsprofile]
+aws_access_key_id = aws-access
+aws_secret_access_key = aws-secret
+`
+	// Create a temporary directory for the dummy aws config
+	tempDirName, err := ioutil.TempDir("", "test")
+	if err != nil {
+		t.Fatal("Error while creating the dummy ecs config directory")
+	}
+	os.Setenv("HOME", tempDirName)
+	os.Setenv("AWS_DEFAULT_REGION", region)
+	defer os.Clearenv()
+	defer os.RemoveAll(tempDirName)
+
+	// save the aws config
+	fileInfo, err := os.Stat(tempDirName)
+	assert.NoError(t, err)
+	mode := fileInfo.Mode()
+	err = os.MkdirAll(tempDirName+"/.aws", mode)
+	assert.NoError(t, err, "Could not create aws config directory")
+	defer os.RemoveAll(tempDirName)
```

Comment:
Maybe silly question, but isn't this a duplicate of line 182?

Reply:
Oh oops, you're right.